src/qos_enclave/Cargo.lock

```diff
@@ -679,6 +1241,15 @@ version = "0.4.3"
 source = "registry+https://github.com/rust-lang/crates.io-index"
 checksum = "7f24254aa9a54b5c858eaee2f5bccdb46aaf0e486a595ed5fd8f86ba55232a70"
 
+[[package]]
+name = "hmac"
```

Comment:
Also not seeing this in the internal review, but that's a "trust external" sort of scenario (RustCrypto project)